app/controllers/cart_controller.rb

```diff
@@ -22,13 +22,11 @@ def destroy
   end
 
   def increment
-    change_quantity(true, @order, params[:book_id])
-    redirect_to cart_path
+    change_quantity(true)
```

Comment:
It's not clear what does the `true` mean. Please use keyword arguments in such cases:
`change_quantity(increment: true)`
https://robots.thoughtbot.com/ruby-2-keyword-arguments